Tests/FluentAssertions.Equivalency.Specs/DataTableSpecs.cs

```diff
@@ -32,6 +32,105 @@ public void When_data_tables_are_both_null_equivalence_test_should_succeed()
         ((DataTable)null).Should().BeEquivalentTo(null);
     }
 
+    [Fact]
+    public void When_row_match_mode_is_invalid_it_should_fail()
+    {
+        // Arrange
+        var typedDataSet1 = CreateDummyDataSet<TypedDataSetSubclass>();
```

Comment:
🤔 Do you really need two datasets here? If not, remove one to not obfuscate the test. If you do need it, please use `subject` and `expectation` as the variable names